File: src/finiteVolume/cellCentred/operators/expression.cpp
// SPDX-FileCopyrightText: 2023 - 2025 NeoN authors
//
// SPDX-License-Identifier: MIT

#include "NeoN/core/parallelAlgorithms.hpp"
#include "NeoN/finiteVolume/cellCentred/expression.hpp"

namespace NeoN::finiteVolume::cellCentred
{

// instantiate the template class
template class Expression<scalar>;
// template class Expression<Vec3>;

};

File: src/neon/blockamr/dsl/expression.py
# SPDX-FileCopyrightText: 2023 - 2026 NeoN authors
#
# SPDX-License-Identifier: MIT

"""Deprecated back-compat shim: Expression is now the unified Equation.

Kept as a thin adapter so old call sites keep working; deleted in plan 06.
"""

from .equation import Equation


class Expression(Equation):
    """Back-compat alias for the unified Equation. Use Equation directly."""
